libqtile/widget/net.py

```diff
@@ -57,25 +57,33 @@ def get_stats(self):
         lines = []  # type: List[str]
         with open('/proc/net/dev', 'r') as f:
             lines = f.readlines()[2:]
-        interfaces = {}
+        interfaces = {'net': {'down': 0, 'up': 0}}
         for s in lines:
             int_s = s.split()
             name = int_s[0][:-1]
             down = float(int_s[1])
             up = float(int_s[-8])
             interfaces[name] = {'down': down, 'up': up}
+            interfaces['net']['down'] += down
+            interfaces['net']['up'] += up
+        # net_up = 0
+        # net_down = 0
+        # for iface in interfaces:
+        #     net_up += interfaces[iface]['up']
+        #     net_down += interfaces[iface]['down']
+        # interfaces['net'] = {'down': down, 'up': up}
         return interfaces
 
     def _format(self, down, up):
         down = "%0.2f" % down
         up = "%0.2f" % up
-        if len(down) > 5:
-            down = down[:5]
-        if len(up) > 5:
-            up = up[:5]
+        if len(down) > 4:
+            down = down[:4]
+        if len(up) > 4:
+            up = up[:4]
```

Comment:
Why are you making this change? If you format a number >100 and < 1000, for example 123.456, you'll end up with something like `123.` here, where the trailing period is odd, rather than `123.4`